app/src/main/java/com/hedvig/app/feature/crossselling/ui/CrossSellingResultActivity.kt

```diff
@@ -9,20 +9,25 @@ import com.hedvig.app.feature.chat.ui.ChatActivity
 import com.hedvig.app.feature.loggedin.ui.LoggedInActivity
 import com.hedvig.app.feature.loggedin.ui.LoggedInTabs
 import org.koin.android.ext.android.inject
+import org.koin.androidx.viewmodel.ext.android.viewModel
+import org.koin.core.parameter.parametersOf
 import java.time.Clock
 import java.time.format.DateTimeFormatter
 
 class CrossSellingResultActivity : BaseActivity() {
-
     private val clock: Clock by inject()
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        val crossSellingResult = intent.getParcelableExtra<CrossSellingResult>(CROSS_SELLING_RESULT)
+    private val tracker: CrossSellTracker by inject()
+    private val crossSellingResult: CrossSellingResult
+        get() = intent.getParcelableExtra(CROSS_SELLING_RESULT)
             ?: throw IllegalArgumentException(
                 "Programmer error: CROSS_SELLING_RESULT not provided to ${this.javaClass.name}"
             )
+    private val model: CrossSellResultViewModel by viewModel { parametersOf(crossSellingResult) }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
 
+        tracker.view(crossSellingResult)
```

Comment:
Now we're doing this here and [here](https://github.com/HedvigInsurance/android/pull/746/files#diff-5d00ecd4c87d651b28a245689421c2c663fdce75ed428074b8516c994f9b2f21R10) at the same time right? It's the same line of code twice. Are we sure we want to have the tracker inside the Activity as well? I thought just in the init() of the VM would be enough? What am I missing? @hedvigoscar